app/models/wpcc/contributions_calendar.rb

```diff
@@ -7,25 +7,31 @@ class ContributionsCalendar
                   :employer_percent
 
     def schedule
-      periods.map do |period|
+      filtered_periods.map do |period|
         Wpcc::PeriodContributionCalculator.new(
           name: period.name,
-          employee_percent: period.employee_percent,
-          employer_percent: period.employer_percent,
+          employee_percent: period.highest_employee_percent,
+          employer_percent: period.highest_employer_percent,
           eligible_salary: eligible_salary,
           salary_frequency: salary_frequency,
           tax_relief_percent: period.tax_relief_percent
         ).contribution
       end
     end
 
+    delegate :periods, to: :period_filter
+
+    def filtered_periods
+      period_filter.filter
+    end
```

Comment:
A minor thing, can we delegate :filtered_periods too? (It is not a blocker though)